packages/lab/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+
+import clamp from "lodash/clamp";
+import { HvTypography } from "@hitachivantara/uikit-react-core";
+
+import { withStyles } from "@material-ui/core";
+
+import styles from "./styles";
+
+/**
+ * ProgressBar provides feedback about a process that is taking place in the application.
+ */
+const HvProgressBar = (props) => {
+  const {
+    className,
+    classes,
+    value = 0,
+    error,
+    undeterminate = false,
+    "aria-label": ariaLabel,
```

Comment:
Explicitly defining the aria-label prop in here is not necessary because it is an standard html prop, so it is implicitly present in the `...others`